### ISSUE TITLE
Normalize data + train_test_split implementation

### ISSUE THREAD

**Issue by Dichik** [Dichik/image-recognition#2]
- implement this method from scratch
